### ISSUE TITLE
Link to signature-algorithms - page does not exist

### ISSUE THREAD

**Issue by lukasz-szewc** [w3c-ccg/http-signatures#40]
Hi,

Inside point `2.1.4.  algorithm` you can find this link : http://www.iana.org/assignments/signature-algorithms with valid values for this parameter. The problem is that this link is showing that page does not exist.

Can anyone provide list of these valid values?

**Comment by lukasz-szewc** [comment]
Does anyone know whether or not there is a new valid URL to this page or alternatively what was the content of that page?

**Comment by msporny** [comment]
> Inside point `2.1.4. algorithm` you can find this link : http://www.iana.org/assignments/signature-algorithms with valid values for this parameter. The problem is that this link is showing that page does not exist.
> Can anyone provide list of these valid values?

That registry won't be created until after the document is a valid RFC... if ever... the registry values will probably be updated to be date-anchored values like "RsaHttpSignature2018" The current valid values are listed here:

https://w3c-dvcg.github.io/http-signatures/#sa-registry

**Comment by aljones15** [comment]
the link actually should be:
https://w3c-ccg.github.io/http-signatures/#hsa-registry

**Comment by ioggstream** [comment]
@msporny signature-algorithms is mentioned here too https://tools.ietf.org/html/draft-pardue-quic-http-mcast-04#section-3.9

**Comment by tpa10** [comment]
Actually, both of the above URLs 
   (https://w3c-dvcg.github.io/http-signatures/#sa-registry and  https://w3c-dvcg.github.io/http-signatures/#hsa-registry)
get 404s when you try to follow them. :(

**Comment by aljones15** [comment]
> Actually, both of the above URLs (https://w3c-dvcg.github.io/http-signatures/#sa-registry and https://w3c-dvcg.github.io/http-signatures/#hsa-registry) get 404s when you try to follow them. :(

this one works: https://w3c-ccg.github.io/http-signatures/#hsa-registry

https://w3c-ccg.github.io/http-signatures/#sa-registry

Looks like their was a typo in my comment that I'm correcting now.

p.s. I believe this version of the spec is old and the current version is the HTTP working group one: https://httpwg.org/http-extensions/draft-ietf-httpbis-message-signatures.html

I might be wrong about the above as I don't work on this spec.

**Comment by msporny** [comment]
> I might be wrong about the above as I don't work on this spec.

You're correct... the HTTML WG has taken over the spec and all work is happening there, now.

Perhaps we should archive this repo or redirect people over there... We were managing implementations here, but perhaps we should manage implementations in the HTTP WG from now on.
